packages/material-ui/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig",
+  "compilerOptions": {
+    "baseUrl": "../../",
+    "paths": {
+      "@material-ui/core": ["packages/material-ui/src"],
```

Comment:
`tsconfigs` are only shallow merged so we have to repeat the `paths` from the extended `tsconfig`